src/app/feed/feed-post/feed-post.component.ts

```diff
@@ -120,6 +149,58 @@ export class FeedPostComponent implements OnInit {
       this.post.RecloutCount = 0;
     }
     this.setEmbedURLForPostContent();
+    if (this.postContent.IsNFT && !this.nftEntryResponses?.length) {
```

Comment:
Thoughts on tradeoffs around just returning this information in/with the PostEntryResponse?

Reply:
So the reason I kept it separate is to keep the response size lower when fetching multiple posts. If someone has a few posts with 1000 copies, it can get to be a bit large. Additionally, others building off the API may not need the NFT details to meet their needs.

However, I can't deny that it would simply things if we included it with every PostEntryResponse.